schemas/documents/contributions/tool.js

```diff
@@ -136,9 +136,9 @@ export default {
               .filter((segment) => segment !== 'blob')
               .join('/');
 
-            // If the person provided only the repository URL, we'll infer the file is master/README.md
-            if (!filePath?.length) {
-              filePath = 'master/README.md';
+            // If the person provided only the repository URL, we'll infer the file is main/README.md
+            if (!filePath) {
+              filePath = 'main/README.md';
```

Comment:
@joneidejohnsen I think this change should remain as was (**though** since we changed this in the website part, any changes here shouldn't affect the front end anymore).

The `master` branch, specifically is a fall back branch for old repos that don't have the main.
New branches that comes with `main` still work if you target the same link with `master` instead.
Old branches that come with `master`will break if you try to fetch from `main`

Reply:
Makes sense – fixed in https://github.com/sanity-io/community-studio/pull/108/commits/1709d6cb8c354e994a047ed3f0726c5df8d4e1f4